google/auth/transport/requests.py

```diff
@@ -588,12 +589,12 @@ def request(
 
         return response
 
-    @property
-    def is_mtls(self):
-        """Indicates if the created SSL channel is mutual TLS."""
-        return self._is_mtls
-
     def close(self):
         if self._auth_request_session is not None:
             self._auth_request_session.close()
         super(AuthorizedSession, self).close()
+
+    @property
+    def is_mtls(self):
+        """Indicates if the created SSL channel is mutual TLS."""
+        return self._is_mtls
```

Comment:
nit: Try to keep whitespace changes out of the diff.

Reply:
addressed.